recipe/meta.yaml

```
@@ -16,10 +16,7 @@ source:
 
 build:
   number: {{ build }}
-  skip: true  # [(win and py==310) or (cuda_compiler_version in (undefined, 'None') and (linux64 or win)) or (cuda_compiler_version != '10.2')]
-  script_env:
-    - CONFIG
-    - CUDA_HOME  # [ppc64le]
+  skip: true  # [(cuda_compiler or "None") == "None"]
```

Suggesting only building for `linux-64` (consistent with what has been done so far)

```suggestion
  skip: true  # [not linux64]
  skip: true  # [(cuda_compiler or "None") == "None"]
```

---

Yes, I'm also in favor of this.